### DISCUSSION TITLE
Placement of Readthedocs widget (aka flyout menu)

### DISCUSSION THREAD

**Discussion by chrisjsewell** [pradyunsg/furo#359] [Ideas] [answered]
Currently, it sits on the bottom right of the screen, and is always visible (even on mobile).
But I believe it is possible to supply a `div` with a certain class in the left ToC bar, then RTD knows to but it there, as per https://sphinx-rtd-theme.readthedocs.io

current:

<img width="160" alt="image" src="https://user-images.githubusercontent.com/2997570/150093892-cb5b75d8-2e52-4a1c-abae-752dec31896c.png">

new:

<img width="331" alt="image" src="https://user-images.githubusercontent.com/2997570/150093983-fb2fa1ca-ec5c-4a40-afbf-c59ab176ef3f.png">

Then it would be hidden on mobile, and not get in the way.

**Comment by chrisjsewell** [comment]
Apparently it's termed the flyout menu: https://docs.readthedocs.io/en/stable/glossary.html?highlight=flyout#term-flyout-menu, but can't remember where I saw how to place it at mo

**Comment by pradyunsg** [comment, marked as answer]
Oh, this got implemented in https://pradyunsg.me/furo/changelog/#jazzy-jasmine

**Reply by chrisjsewell** [reply]
Ah didn't notice that, nice one cheers!

**Reply by pradyunsg** [reply]
https://github.com/pradyunsg/furo/blob/main/src/furo/theme/furo/sidebar/rtd-versions.html
https://github.com/pradyunsg/furo/blob/main/src/furo/assets/styles/extensions/_readthedocs.sass

These are the two main things for this, if I remember correctly.
